Python/Tests/IpcJsonTests/IpcJsonTests.cs

```diff
@@ -256,6 +256,8 @@ public class IpcJsonTests {
         }
     }
 
+#pragma warning disable 0649
```

Comment:
What warning is this? Can you add a comment with the warning message as well, otherwise every time I see it I'm going to go look it up :)